crates/providers/src/anilist/mod.rs

```diff
@@ -1,3 +1,4 @@
+use std::fmt::Display;
 use anyhow::{anyhow, Result};
```

Comment:
Add a new line here. I prefer to keep `std` imports separately.

Reply:
Resolved I disabled my formatter since the default settings seem to be different than what your settings are in weird edge cases ill keep it enabled and inspect from here on.